crates/holochain/src/conductor/metrics.rs

```diff
@@ -1,23 +1,8 @@
 use opentelemetry_api::{global::meter_with_version, metrics::*};
 
-pub type P2pEventDurationMetric = Histogram<f64>;
 pub type PostCommitDurationMetric = Histogram<f64>;
 
-#[allow(dead_code)]
-pub fn create_p2p_event_duration_metric() -> P2pEventDurationMetric {
```

Comment:
We'd removed this when getting rid of the p2p even dispatch thing. That doesn't have an exact equivalent but we do have request-reply in `holochain_p2p` so I've moved it to there